composer.json

```diff
@@ -25,6 +25,10 @@
     "mautic/core-lib": "^5.0"
   },
   "require-dev": {
+    "codeception/codeception": "^4.2",
+    "codeception/module-asserts": "^2.0",
+    "codeception/module-db": "^2.1",
+    "codeception/module-webdriver": "^2.0",
```

Comment:
We can introduce namespace for the tests.

We can add something like
```
"autoload-dev": {
        "psr-4": {
            "Mautic\\Tests\\": "tests/"
        }
    },
```
right below following snippet.
```
  "autoload": {
    "psr-4": {
      "MauticPlugin\\": "plugins/"
    }
  },
```

Once we do that, we can namespace all the tests in the `/tests` directory.

Reply:
Example https://github.com/Codeception/symfony-demo/blob/master/composer.json#L72C5-L76C7 

After adding those, you can notice any codeception test with `App\Tests\`

In our case, we can replace `App` with `Mautic`